src/View/HalJsonStrategy.php

```diff
@@ -32,7 +32,7 @@ class HalJsonStrategy extends JsonStrategy
      */
     public function __construct(HalJsonRenderer $renderer)
     {
-        $this->renderer = $renderer;
+        parent::__construct($renderer);
```

Comment:
It's probably safer to leave this as it was. While the `HalJsonRenderer` *currently* extends the `JsonRenderer`, that's an implementation detail, and could change in the future.